src/components/QueryEditor/EditorStack.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Stack } from '@grafana/ui';
+import { ThemeSpacingTokens } from '@grafana/data';
+
+export type Direction = 'row' | 'row-reverse' | 'column' | 'column-reverse';
+export type AlignItems =
+  | 'stretch'
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'baseline'
+  | 'start'
+  | 'end'
+  | 'self-start'
+  | 'self-end';
+
+interface StackProps {
+  children?: React.ReactNode;
+  direction?: Direction;
+  alignItems?: AlignItems;
+  wrap?: boolean;
+  gap?: ThemeSpacingTokens;
+}
+
+export const EditorStack = ({ children, wrap: wrapItems, ...props }: StackProps) => {
+  return (
+    <Stack
+      wrap={(wrapItems ?? 'true') ? 'wrap' : undefined}
```

Comment:
```suggestion
      wrap={wrapItems ? 'wrap' : undefined}
```

I believe we can simplify this and it is going to evaluate the same.

Reply:
We want it to resolve to true if it's undefined too. However, I did manage to simplify it by giving it a default value in the props list